### PR TITLE
feat/new-header

### PR DESCRIPTION
**Pull request by brian-mulier-p**
closes https://github.com/kestra-io/kestra/issues/706 
closes https://github.com/kestra-io/kestra/issues/2244

### CODE REVIEW

**Comment by anna-geller** [comment]
Can you add a screenshot @brian-mulier-p?

**Comment by brian-mulier-p** [comment]
image
image

Note that there is some mismatch with Figma as we have to rework colors not to require exceptions specifically for the header, colors should be synced between Figma & UI so the colors match naturally. I reverted any exception around it after asking @tchiotludo how to handle it.

**Comment by anna-geller** [comment]
Nice! we need to add one more option here "Execute with labels"
image

And cut the labels here:

image

**Comment by brian-mulier-p** [comment]
I think that's another subject :+1: Can you create an issue if there is none ? This PR will already be huge so it shouldn't be  included it there

**Comment by anna-geller** [comment]
assigned to you :) https://github.com/kestra-io/kestra/issues/2173

File: ui/src/components/flows/TriggerFlow.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
     <div class="trigger-flow-wrapper">
-        <el-button class="edit-flow-trigger-button" :icon="icon.Flash" :disabled="disabled || flow?.deleted" size="large" :type="type" @click="onClick">
+        <el-button class="edit-flow-trigger-button" :icon="icon.Flash" :disabled="disabled || flow?.deleted" :type="type" @click="onClick">
             {{ $t('New execution') }}
```
Comment (anna-geller) [inline ui/src/components/flows/TriggerFlow.vue:4]:
```suggestion
            {{ $t('Execute') }}
```

**Comment by anna-geller** [comment]
Discussed some change requests via Slack:

1. Move the `"Successfull saved"` popup to the bottom as it currently covers the Execute button.
2. In a separate issue, we will tackle the Execute button to allow `Execute with labels` and only show the popup to add Execution labels when user clicks on that second option to execute with labels -- see issue https://github.com/kestra-io/kestra/issues/2173#issuecomment-1764390375 
3. The Import and Source Search both need to be added as independent actions in the Flows menu rather than hidden behind Actions

currently is:
image

should be:
image

4. Also for the Flows page - when you open the Editor tab, the Save and Actions menu should ideally be added to the main menu if possible rather than within the Editor tab (might also be easier with VS Code later on - WDYT @brian-mulier-p?)

image

5. For all tabs within the Executions page, there menus should also be directly visible rather than hidden behind Actions:

currently is:
image

should be:
image

**Review by anna-geller — CHANGES_REQUESTED** [review]
Discussed some change requests via Slack:

- [x] Move the `"Successfull saved"` popup to the bottom as it currently covers the Execute button.
- [x] In a separate issue, we will tackle the Execute button to allow `Execute with labels` and only show the popup to add Execution labels when user clicks on that second option to execute with labels -- see issue https://github.com/kestra-io/kestra/issues/2173#issuecomment-1764390375 
- [x] The Import and Source Search both need to be added as independent actions in the Flows menu rather than hidden behind Actions

currently is:
image

should be:
image

- [x] Also for the Flows page - when you open the Editor tab, the Save and Actions menu should ideally be added to the main menu if possible rather than within the Editor tab (might also be easier with VS Code later on - WDYT @brian-mulier-p?)

image

- [x]  For all tabs within the Executions page, there menus should also be directly visible rather than hidden behind Actions:

currently is:
image

should be:
image

**Comment by anna-geller** [comment]
list of follow-up issues that are out-of-scope in this PR https://github.com/kestra-io/kestra/issues/2322

**Review by Skraye — CHANGES_REQUESTED** [review]
Missing space when clicking source search: 
image

Flow view > Components seems a bit too big and we have a useless scroll (appear on all tab that goes to bottom):
image

Should be same colors, no hovers on 2 first icons
image

You can use the shortcut, write, but not select with enter, you must click, so feature become useless:
image

execution page : 
image

Arriving on blueprint page when you had already search something:
image

**Comment by brian-mulier-p** [comment]
@anna-geller
4. Discussed with Ludo already, hierarchically there is no way without big hack to do what's asked that's why those buttons were added there: it is way too bound to the editor tab (you can edit the source code from those buttons). Idk when and how we will tackle but this is the proper way to do it right now (seen with @Nico-Kestra & @tchiotludo).
5. Please confirm with @tchiotludo, I've been asking what to do and he told me at first to keep only 1 or 2 main buttons and add a dropdown for further actions to prevent having to handle overflow and lower res. as it's a bit complex from @Nico-Kestra design (the 3 dots only on lower res would imply a lot of duplication or a big hack).

@Skraye
will look at all but:

> You can use the shortcut, write, but not select with enter, you must click, so feature become useless:
This is intended, right now you have a fake search which will later be extended to executions & flows. This will be tackled when it will be done as it will then provide a true value. For now this fake search is kind of a placeholder so I didn't take time to handle this keyboard-only navigation

**Review by anna-geller — APPROVED** [review]
Great work! all issues addressed ✅ 

image

**Comment by brian-mulier-p** [comment]
Just found out that the welcome screen looks ugly as we don't have any header and by default there is no margin, will fix that

**Comment by Skraye** [comment]
Small issue :
image

Those two buttons do the same things; maybe one should be removed? 
image

**Comment by anna-geller** [comment]
> Small issue : image
> 
> Those two buttons do the same things; maybe one should be removed? image

Great point Yann. We did that deliberately for 2 reasons:
1. For the SaaS version, Live help can point to a different channel in the future e.g. AI chatbot
2. Emmanuel considered it important to show users they can get unblocked and get help when they are stuck.

let's keep both but good observation 👍

**Comment by brian-mulier-p** [comment]
I asked the same thing to Nico :P

**Review by Skraye — APPROVED** [review]
trivial matter, LGTM

**Comment by Nico-Kestra** [comment]
Capture d’écran 2023-10-17 à 18 21 11

File: ui/src/components/executions/ExecutionRoot.vue [hunk 2]
```diff
@@ -1,43 +1,59 @@
...
                         </el-button>
-                    </a>
-                </li>
-                <li>
-                    <el-button :icon="Delete" size="large" type="default" v-if="canDelete" @click="deleteExecution">
-                        {{ $t('delete') }}
-                    </el-button>
+                        <template #dropdown>
+                            <el-dropdown-menu class="m-dropdown-menu">
+                                <a class="el-dropdown-menu__item d-flex gap-2" :href="`${finalApiUrl}/executions/${execution.id}`" target="_blank">
```
[the remark this answers is not in the document]
Comment (brian-mulier-p) [reply ui/src/components/executions/ExecutionRoot.vue:13]: image
No that's a wider dropdown issue (already present). Will not fix it there, we have to apply some styling to dropdown to handle it globally

**Comment by brian-mulier-p** [comment]
> Just found out that the welcome screen looks ugly as we don't have any header and by default there is no margin, will fix that

Fixed

**Comment by Nico-Kestra** [comment]
Capture d’écran 2023-10-18 à 08 59 13
Shift the app so that the purple band is not above the menu.

**Pull request merged**